specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/Operations_List.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
All of the examples here are caches. I don't know if that matters. (I'm happy enough to resolve this if the answer is "Yeah, that doesn't matter.")

Reply:
Added an amlFilesystem to the values.